react-components/src/components/Reveal3DResources/hooks/useCalculateCadStyling.test.tsx

```diff
@@ -136,6 +123,28 @@ describe(useCalculateCadStyling.name, () => {
     });
   });
 
+  test('returns empty style groups when no mappings exist', async () => {
```

Comment:
I have moved this test from further below in the file; I think it makes more sense to have it up here